src/app/cart/CartSummary.tsx

```diff
@@ -20,14 +20,19 @@ export type WithCheckoutCartSummaryProps = {
 const CartSummary: FunctionComponent<WithCheckoutCartSummaryProps> = ({
     cartUrl,
     ...props
-}) => (
-    withRedeemable(OrderSummary)({
-        ...props,
-        cartUrl,
-        headerLink: (
-            <EditLink url={ cartUrl } />
-        ),
-    })
-);
+}) => {
+    // if there's a sub path name in checkout, assume it's a buy now cart
+    const isBuyNowCart = window.location.pathname.replace('/checkout', '').length > 0;
```

Comment:
In addition to standard checkout, we have [embedded checkout](https://www.mody.com.au/embedded-checkout) for Wordpress intergration. This line treats embedded checkout as if it were a buy now cart checkout. Think we also need to rule out the `/embedded-checkout` case.